deploy/setup-project.sh

```diff
@@ -1,13 +1,26 @@
 #!/bin/bash
 
+# This script will setup the given project with a Service Account that has the correct
+# restricted permissions to run the gcp_compute_persistent_disk_csi_driver and download
+# the keys to a specified directory
+
+# WARNING: This script will delete and recreate all service accounts, bindings, and keys.
+# Great care must be taken to not run the script with a service account that is currently
+# in use.
+
+# Args:
+# PROJECT: GCP project
+# GCE_PD_SA_NAME: Name of the service account to create
+# GCE_PD_SA_DIR: Directory to save the service account key
```

Comment:
Are there defaults set?

Reply:
Should there be defaults? I think these are all important enough that there shouldn't be any defaults. I have `set -o nounset` so it throws an error if the vars are unset when the script runs